RecoTauTag/Configuration/python/HPSPFTaus_cff.py

```diff
@@ -128,6 +129,9 @@
     Prediscriminants = requireDecayMode.clone()
 )
 hpsPFTauDiscriminationByLooseCombinedIsolationDBSumPtCorr.qualityCuts.isolationQualityCuts.minTrackPt = 0.5
+phase2_common.toModify(hpsPFTauDiscriminationByLooseCombinedIsolationDBSumPtCorr.qualityCuts,
+                       isolationQualityCuts = dict( minTrackPt = cms.double(0.8) )
```

Comment:
It is not fundamental, but it is usually more convenient (and safer in case of possible typing mistakes) to remove type specification when updating a parameter via modifier, e.g.
```
                       isolationQualityCuts = dict( minTrackPt = 0.8 )
```